### PR TITLE
Add support for Cloud Hypervisor

### PR DESCRIPTION
**Pull request by sboeuf**
This patchset adds Cloud Hypervisor support to OVMF binary for x86 architecture.

### CODE REVIEW

File: OvmfPkg/AcpiPlatformDxe/AcpiPlatformDxe.inf [hunk 1]
```diff
@@ -54,6 +55,8 @@
 
 [Pcd]
   gEfiMdeModulePkgTokenSpaceGuid.PcdPciDisableBusEnumeration
+  gUefiOvmfPkgTokenSpaceGuid.PcdXenPvhStartOfDayStructPtr
+  gUefiOvmfPkgTokenSpaceGuid.PcdXenPvhStartOfDayStructPtrSize
```
Comment (rbradford) [inline OvmfPkg/AcpiPlatformDxe/AcpiPlatformDxe.inf:59]: Is this change necessary?

**Review by rbradford — APPROVED** [review]

File: OvmfPkg/AcpiPlatformDxe/AcpiPlatformDxe.inf (hunk 1 above)
Comment (sboeuf) [reply OvmfPkg/AcpiPlatformDxe/AcpiPlatformDxe.inf:59]: Not at all! It's legacy from Xen.

**Comment by sboeuf** [comment]
@jyao1 @kraxel just curious to know what are the next steps in order to get the patch series merged (I mean now that you both reviewed it)?

**Comment by kraxel** [comment]
> @jyao1 @kraxel just curious to know what are the next steps in order to get the patch series merged (I mean now that you both reviewed it)?

Now that the uncustify reformating patches seem to be landed the series needs reformating (see uncrustify docs, there is a longish git filter-branch command line which should handle this automatically, didn't try that yet but will probably do that tomorrow for my pending patches).  Then repost the reformated patches and they should go in without too much trouble.

**Comment by sboeuf** [comment]
I rebased the PR.
